tasks.py

```diff
@@ -299,9 +299,9 @@ def _replace(pattern):
         newpat = pattern.format("v{}").format(version)
         run("perl -pi -e 's#{}#{}#g' website/content/*.md website/content/*/*.md".format(oldpat, newpat),
             echo=True)
-    _replace("/google/metallb/{}")
-    _replace("/google/metallb/tree/{}")
-    _replace("/google/metallb/blob/{}")
+    _replace("/metallb/metallb/{}")
+    _replace("/metallb/metallb/tree/{}")
+    _replace("/metallb/metallb/blob/{}")
```

Comment:
Have you created a release locally to test this? I mean, test that this replace works as expected and there is no missing URL that isn't replaced?

Reply:
I wasn't sure it was safe to run the release logic. I'll give it a try.

Reply:
This turned out to be tricky to check/fix. I keep getting unexpected results. Will update once I manage to figure it out.

Reply:
Looks like this works fine.